tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp

```diff
@@ -302,6 +302,196 @@ void test_bulged_frustum_inv_map() {
   test_inverse_map(map, test_point3);
   test_inverse_map(map, test_point4);
 }
+
+void test_bulged_frustum_equiangular_full() {
+  INFO("Bulged frustum jacobian");
+  const std::array<std::array<double, 2>, 4> face_vertices{
+      {{{-2.0, -2.0}}, {{2.0, 2.0}}, {{-4.0, -4.0}}, {{4.0, 4.0}}}};
+  const CoordinateMaps::Frustum map(
+      face_vertices, 2.0, 4.0, OrientationMap<3>{}, true, 1.0, false, 1.0, 0.0);
+
+  // Points on the upper +zeta face:
+  const std::array<double, 3> lowerxilowereta{{-1.0, -1.0, 1.0}};
+  const std::array<double, 3> zeroxilowereta{{0.0, -1.0, 1.0}};
+  const std::array<double, 3> upperxilowereta{{1.0, -1.0, 1.0}};
+
+  const std::array<double, 3> lowerxizeroeta{{-1.0, 0.0, 1.0}};
+  const std::array<double, 3> zeroxizeroeta{{0.0, 0.0, 1.0}};
+  const std::array<double, 3> upperxizeroeta{{1.0, 0.0, 1.0}};
+
+  const std::array<double, 3> lowerxiuppereta{{-1.0, 1.0, 1.0}};
+  const std::array<double, 3> zeroxiuppereta{{0.0, 1.0, 1.0}};
+  const std::array<double, 3> upperxiuppereta{{1.0, 1.0, 1.0}};
+
+  // If Frustum is successfully bulged, points on upper +zeta face
+  // should have the same radius:
+  CHECK(magnitude(map(lowerxilowereta)) == approx(4.0 * sqrt(3.0)));
+  CHECK(magnitude(map(upperxilowereta)) == approx(4.0 * sqrt(3.0)));
+  CHECK(magnitude(map(lowerxiuppereta)) == approx(4.0 * sqrt(3.0)));
+  CHECK(magnitude(map(upperxiuppereta)) == approx(4.0 * sqrt(3.0)));
+
+  // If Frustum is successfully equiangular, distances equally spaced
+  // in logical space correspond to equal angular distances. For a map
+  // where a logical axis subtends an angle of theta, the jacobian
+  // is radius * theta / 2 at any point along this axis.
+  const std::array<double, 3> lowerxizeroeta_dx_dxi{
+      {map.jacobian(lowerxizeroeta).get(0, 0),
+       map.jacobian(lowerxizeroeta).get(1, 0),
+       map.jacobian(lowerxizeroeta).get(2, 0)}};
+  const std::array<double, 3> zeroxizeroeta_dx_dxi{
+      {map.jacobian(zeroxizeroeta).get(0, 0),
+       map.jacobian(zeroxizeroeta).get(1, 0),
+       map.jacobian(zeroxizeroeta).get(2, 0)}};
+  const std::array<double, 3> upperxizeroeta_dx_dxi{
+      {map.jacobian(upperxizeroeta).get(0, 0),
+       map.jacobian(upperxizeroeta).get(1, 0),
+       map.jacobian(upperxizeroeta).get(2, 0)}};
+  CHECK(magnitude(lowerxizeroeta_dx_dxi) == approx(4.0 * M_PI_4 * sqrt(3.0)));
```

Comment:
Could you define a variable for this value please, and comment or make clearer (e.g. by using a `radius` variable) why this is the expected value? Same for the tests below.